pages/existing_instrumentations.md

```diff
@@ -12,8 +12,20 @@ The following libraries exist to provide instrumentation on various platforms.
 Please refer to their individual documentation for setup and configuration
 guides.
 
+### openzipkin libraries
+
+The following libraries are supported by the OpenZipkin authors.
```

Comment:
Made it into this:

The following libraries are supported by the OpenZipkin authors and are hosted at the [OpenZipkin Github](https://github.com/openzipkin/) group. You can reach out to the authors at the [Zipkin Gitter](https://gitter.im/openzipkin/zipkin/) chat.